frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleBusinessManagerPage/components/AddAbmModal/AddAbmModal.tsx

```diff
@@ -135,14 +134,7 @@ const AddAbmModal = ({ onCancel, onAdded }: IAddAbmModalProps) => {
           graphicName={"file-p7m"}
           buttonType="link"
           buttonMessage={isUploading ? "Uploading..." : "Upload"}
-          filePreview={
-            tokenFile && (
-              <FileDetails
-                details={{ name: tokenFile.name }}
-                graphicName="file-p7m"
-              />
-            )
-          }
+          fileDetails={tokenFile ? { name: tokenFile.name } : undefined}
```

Comment:
Looks like the graphic name may have been accidentally removed here?

Reply:
It's passed in from line 134